src/app/public/modules/sky-tree-view/sky-tree-view-toolbar.component.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  ViewEncapsulation
+} from '@angular/core';
+
+import {
+  TreeModel,
+  TreeNode
+} from 'angular-tree-component';
+
+import {
+  SkyTreeViewOptions
+} from './types/sky-tree-view-options';
+
+@Component({
+  selector: 'sky-tree-view-toolbar',
+  templateUrl: './sky-tree-view-toolbar.component.html',
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SkyTreeViewToolbarComponent {
+
+  @Input()
+  public treeModel: TreeModel;
```

Comment:
Instead of passing in the entire tree model, could we only set values that the toolbar needs, specifically?

Reply:
Moot now - see previous comment.